pulsar-client-api/src/main/java/org/apache/pulsar/common/api/EncryptionContext.java

```diff
@@ -39,6 +39,7 @@ public class EncryptionContext {
     private CompressionType compressionType;
     private int uncompressedMessageSize;
     private Optional<Integer> batchSize;
+    private boolean isEncrypted;
```

Comment:
It's better to add a doc to explain this field.

Reply:
Add @RobertIndie